tests/system/Commands/Database/ShowTableInfoTest.php

```diff
@@ -64,11 +64,11 @@ public function testDbTable(): void
         $expected = 'Data of Table "db_migrations":';
         $this->assertStringContainsString($expected, $result);
 
-        $expected = <<<'EOL'
-            +----+----------------+--------------------+-------+---------------+------------+-------+
-            | id | version        | class              | group | namespace     | time       | batch |
-            +----+----------------+--------------------+-------+---------------+------------+-------+
-            EOL;
+        $expected = implode(PHP_EOL, [
+            '+----+----------------+--------------------+-------+---------------+------------+-------+',
+            '| id | version        | class              | group | namespace     | time       | batch |',
+            '+----+----------------+--------------------+-------+---------------+------------+-------+',
+        ]);
```

Comment:
Writing this `$expected` is a bit pain. 
The reason to use `<<<'EOL'` is that we want to just copy and paste the output.

The newline code in the source is always `\n`.
Why don't you add a method to convert newline code?

Reply:
Thanks! I myself is struggling with these `implode()` calls. 😂